content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+---
+title: Jenkins Configuration as Code (JCasC)
+layout: section
+wip: true
+---
+
+The Jenkins Configuration as Code (JCasC) feature defines Jenkins configuration parameters
+in a human-readable YAML file that can be stored under source code.
+This essentially captures the configuration parameters and values that are used when configuring Jenkins from the web UI.
+The configuration can then be modified by editing this file and then applying it.
+
+Configuring and tuning Jenkins and its plugins is a complex process,
+with dozens of parameters to set within the web UI `manage` section.
+Traditionally, experienced Jenkins administrators
+created Apache Groovy `init` scripts
+to automate the configuration for their Jenkins instances.
+This works but requires in-depth understanding of Jenkins APIs
+and the ability to write Groovy scripts.
+Such scripts are powerful and can do almost anything,
+but they also provide few protections against configuration errors.
+
+JCasC provides the convenience and flexibility
+of configuring controllers without using the UI.
+It does not require more understanding of the configuration parameters
+than is required to configure Jenkins through the UI
+and it provides some checks on the values that are provided.
+
+The JCasC configuration file can be checked into an SCM,
+which enables you to determine who made what modifications to the configuration
+and to roll back to a previous configuration if necessary.
+
+The
+link:https://plugins.jenkins.io/configuration-as-code[Configuration as Code] plugin
+must be installed on the Jenkins controller that you will use to build out your JCasC configuration.
+This plugin is not currently included in the suggested plugins list for Jenkins
+but the Docker image in the
```

Comment:
the docker image doesn't contain it but the helm chart does

Reply:
Perhaps it should, but that's a different discussion 🗡️

Reply:
Maybe I'm being too specific here anyhow.  I'm just going to say that the plugin must be installed and let it go at that -- details about how that happened are actually irrelevant, aren't they?

Reply:
No good reason to go down this rabbit hole in this document, right?  Check "Manage Jenkins" -- if you don't see Configuration as Code", you need to install it.  That seems like all we need to say.  Done.